src/Themes/NexusMods.Themes.NexusFluentDark/Styles/UserControls/WorkspaceSystem/PanelViewStyles.axaml

```diff
@@ -45,14 +66,8 @@
                         <Setter Property="BorderBrush" Value="{StaticResource StrokeTranslucentWeak}" />
                     </Style>
 
-                    <!-- selected tab -->
-                    <Style Selector="^ Border#Container.Selected">
-                        <Setter Property="BorderThickness" Value="1 0 0 0" />
-                    </Style>
-
-                    <!-- not selected tab -->
-                    <Style Selector="^ Border#Container:not(.Selected)">
-                        <Setter Property="BorderThickness" Value="1 0 0 1" />
+                    <Style Selector="^ Border#Container">
```

Comment:
Apparently this targets all Borders called "Container", even from child controls.
It seems we need to be more specific with with our naming or use more specific pathing